components/service/firefox-accounts/src/main/java/mozilla/components/service/fxa/Types.kt

```diff
@@ -235,8 +238,8 @@ fun IncomingDeviceCommand.into(): mozilla.components.concept.sync.DeviceCommandI
 
 fun IncomingDeviceCommand.TabReceived.into(): mozilla.components.concept.sync.DeviceCommandIncoming.TabReceived {
     return mozilla.components.concept.sync.DeviceCommandIncoming.TabReceived(
-        from = this.from?.into(),
-        entries = this.entries.map { it.into() }
+        from = this.sender?.into(),
+        entries = this.payload.entries.map { it.into() }
```

Comment:
The field names here were different between the Rust code and the hand-written Kotlin wrapper, and now that things are autogenerated they're the same.